completion_aggregator/settings/common.py

```diff
@@ -15,3 +15,4 @@ def plugin_settings(settings):
         'sequential',
         'vertical',
     }
+    settings.COMPLETION_AGGREGATOR_ASYNC_AGGREGATION = True
```

Comment:
Not sure about defaulting to `True` here..  Kinda depends on the next  steps planned.
* Are you going to submit a configuration PR to add a cron job to run this new management command if configured?
  Would make sense to try to use a sensible default behaviour that depends on the value of the `COMPLETION_AGGREGATOR_ASYNC_AGGREGATION` LMS env variable.
* If no configuration PR is planned, I think this should default to `False`.

Reply:
Defaulting to True means that Aggregator calculatation won't happen until the value is explicitly set.  This was intentional, to prevent accidental resource hogging while we experiment with how well this performs.  At the same time, StaleCompletions (which don't require any calculation) will still get created, so when Aggregation does get turned on, we won't have missed work in the interim.  This is particularly going to be useful for migrating the progress data to the new system.

Reply:
On the other hand, defaulting to False means that the app is more easily usable in new small installations, and we can disable it explicitly for McKinsey.  So I will follow your suggestion.